spec/requests/claims_spec.rb

```diff
@@ -183,17 +183,20 @@ def check_slug_redirection
     context "when a claim is already in progress" do
       let(:in_progress_claim) { Claim.by_policy(StudentLoans).order(:created_at).last }
 
-      before { start_student_loans_claim }
+      before {
+        start_student_loans_claim
+        set_slug_sequence_in_session(in_progress_claim, "qts-year")
+      }
 
       it "updates the claim with the submitted form data" do
         put claim_path(StudentLoans.routing_name, "qts-year"), params: {claim: {eligibility_attributes: {qts_award_year: "on_or_after_cut_off_date"}}}
 
+        expect(response).to redirect_to(claim_path(StudentLoans.routing_name, "claim-school"))
         expect(in_progress_claim.eligibility.qts_award_year).to eq "on_or_after_cut_off_date"
```

Comment:
@kenfodder I'm not sure why this `put` no longer works. I added the expectaion for the redirect to confirm it is still redirecting correctly in this scenario.

This expectation currently fails:

```ruby
expect(in_progress_claim.eligibility.qts_award_year).to eq "on_or_after_cut_off_date"
```

Reply:
Let me have a look...

Reply:
@mark-pkgflow Needs a `.reload`

```
expect(in_progress_claim.reload.eligibility.qts_award_year).to eq "on_or_after_cut_off_date"
```

Because `in_progress_claim` is called in the before block it's stale when it hit the assertion.